apps/designer/app/designer/features/sidebar-left/panels/pages/pages.tsx

```diff
@@ -129,14 +183,19 @@ export const TabContent = (props: TabContentProps) => {
   const [project] = useProject();
 
   const navigate = useNavigate();
-  const handleSelect = (pageId: string) => {
+  const handleSelect = (pageId: "home" | Page["id"]) => {
     if (project === undefined) {
       return;
     }
-    navigate(`${props.config.designerPath}/${project.id}?pageId=${pageId}`);
+    if (pageId === "home") {
+      navigate(`${props.config.designerPath}/${project.id}`);
+    } else {
+      navigate(`${props.config.designerPath}/${project.id}?pageId=${pageId}`);
```

Comment:
do this path building logic anywhere else? if yes we should have a reusable function

Reply:
What if instead of `config` that we pass around, we'll have something like `shared/routes-paths.ts` with:

```
export const designerPath({projectId, pageId}: {projectId: string, pageId?: string}) => {
  if (pageId) {
    return `/designer/${projectId}?pageId=${pageId}`
  }
  return `/designer/${projectId}`
}

// same for every path that we need in the app...
```

1. Wouldn't have to pass around config
2. Builds the entire path
3. If we need paths to differ in different deployments we can have conditions like `process.env.NODE_ENV === ...` in this file

Reply:
Also found this https://github.com/yesmeck/remix-routes . But not sure about it:

1. Adds another build step (https://github.com/yesmeck/remix-routes#setup)
2. Works only for Remix routes, but we won't necessarily use Remix for e.g. canvas (or it can be Remix, but a separate app)
3. Not sure it will support all our use cases even for Remix routes as well as custom written functions

Reply:
1. yeah path-utils
2. we don't actually need to pass around config, its static, we can import it directly

Reply:
remix-routes package - yeah not sure about it

Reply:
Will do a separate PR: https://github.com/webstudio-is/webstudio-designer/issues/389